gtdynamics/optimizer/EqualityConstraint.h

```diff
@@ -134,6 +135,39 @@ class VectorExpressionEquality : public EqualityConstraint {
   size_t dim() const override;
 };
 
+/** Equality constraint that force factor error to be 0. */
+class FactorEquality : public EqualityConstraint {
```

Comment:
`FactorEquality` is a bit misleading. At first glance it looked like it is comparing the result/errors of two factors. Maybe `ZeroEqualityFactor` or `ZeroEqualityConstraint`?

Reply:
Renamed as `ZeroErrorFactorEquality`